app/state_machines/configurable_state_machine/machine.rb

```diff
@@ -114,7 +114,7 @@ def respond_to_missing?(method, include_private = false)
     end
 
     def event_name(event)
-      if events.include?(event.to_sym)
+      if events.include?(event.to_sym) || event == @kase.class::CREATE_EVENT
```

Comment:
Sorry! One last thing that occurs to me - why isn't our event in the `events` already? Seems like we should be adding it there, not adding it to this conditional as a special case. 

The state machine is one of the bits I think we should use "as intended" - did you manage to see where the events collection is set?